UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -193,13 +193,31 @@ <h4>Metadata techniques</h4>
 			
 			<section id="visual-adjustments">
 				<h3>Visual Adjustments</h3>
+				<!--gautier-->
+				<p>A Yes/No/Unknown answer to the question "Can a user change the aspect of textual contents". </p>
+
+				<p>Indicates if the appearance of the text and the page layout can be modified according to the possibilities offered by the reading system.</p>
 				
-				<p>TBD</p>
-				
-				<p>TBD</p>
+				<p>Readers with visual impairment or cognitive disabilities rely on the ability to change color of background and texts,  font family and font size as well as spaces between letters, words, sentences or paragraphs.</p>
+				<p><!--this one might be moved to the technics-->The reflowable information alone is not sufficient to make sure that modifications operated will not result on text overlaying or unreadable colors.</p>
+			
 				<h4>Examples</h4>
-                <p>TBD</p>
-                <h4>Metadata techniques</h4>
+                <p>The exemples are provided as long and short sentence for flexibility.</p> 
+				 <ul> 
+					 <li>
+				<p>Long sentences:</p>
+				<aside class="example">Appearance of the text and page layout (color of background and texts, font family and font size as well as spaces between letters, words, sentences or paragraphs) can be modified according to the possibilities offered by the reading system.</aside>
+				<p>or</p>
+				<aside class="example">Text and page layout are not adaptable, the reading experience is close to a print version. Reading system can still provide zooming options.</aside>
```

Comment:
The text changes from talking about the text/layout being modifiable to it being adaptable here, which I think can get confusing. Modifiable sound like the better terminology, as saying it's not adaptable sounds like you can modify the text/layout but there's no telling what will happen.